addons/calico-vxlan/calico-vxlan.yaml

```diff
@@ -1263,12 +1364,22 @@ spec:
                   Felix makes reports. [Default: 86400s]'
                 type: string
               useInternalDataplaneDriver:
+                description: UseInternalDataplaneDriver, if true, Felix will use its
+                  internal dataplane programming logic.  If false, it will launch
+                  an external dataplane driver and communicate with it over protobuf.
                 type: boolean
               vxlanEnabled:
+                description: 'VXLANEnabled overrides whether Felix should create the
+                  VXLAN tunnel device for VXLAN networking. Optional as Felix determines
+                  this based on the existing IP pools. [Default: nil (unset)]'
                 type: boolean
               vxlanMTU:
-                description: 'VXLANMTU is the MTU to set on the tunnel device. See
-                  Configuring MTU [Default: 1440]'
+                description: 'VXLANMTU is the MTU to set on the IPv4 VXLAN tunnel
+                  device. See Configuring MTU [Default: 1410]'
```

Comment:
It appears that the default MTU has been changed in Calico 3.23. Do we need to take any action on this?

Reply:
I don't think so.

Reply:
I wonder why they did it

Reply:
It's about ipv6 support: https://github.com/projectcalico/calico/commit/009498285c7f6634ab0fb3db7b38c4567f4cbe34

So probably we can do the same

Reply:
To me this actually isn't a change for us - it is a change in the `FelixConfiguration` CRD, that we are not using, as we do not deploy Calico via operator. In this manifest-based deployment of Calico, there is no use of `FelixConfiguration` at all, and the MTU is passed from the `veth_mtu` config in `calico-config` configmap. By default is set to `0` - which means auto-configuration - and that should work for most of the deployments.